modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java

```diff
@@ -284,7 +285,9 @@ public Redirect getRedirect(
 
 				if (Validator.isNotNull(i18nLanguageId) &&
 					!LanguageUtil.isAvailableLocale(
-						group.getGroupId(), i18nLanguageId)) {
+						group.getGroupId(), i18nLanguageId) &&
+					(!PortalUtil.isControlPanelPath(path) ||
```

Comment:
@joseluisbango Please use `portal.isControlPanelPath` here. The `portal` is already a `@Reference` in this class (all its sub-classes are OSGi components).

Reply:
Also in https://github.com/liferay-core-infra/liferay-portal/pull/961/commits/7bb6c9480ad81ffbbf19601dd57b3ee92764124a